lib/driver.js

```diff
@@ -249,12 +249,21 @@ class AndroidDriver extends BaseDriver {
 
     if (this.opts.disableWindowAnimation) {
       if (await this.adb.isAnimationOn()) {
-        if (await this.adb.getApiLevel() >= 28) { // API level 28 is Android P
+        const apiLevel = await this.adb.getApiLevel();
+
+        if (apiLevel >= 28) { // API level 28 is Android P
           // Don't forget to reset the relaxing in delete session
           log.warn('Relaxing hidden api policy to manage animation scale');
           await this.adb.setHiddenApiPolicy('1');
         }
 
+        // Granting android.permission.SET_ANIMATION_SCALE is necessary to handle animations under API level 26
+        // Read https://github.com/appium/appium/pull/11640#issuecomment-438260477
+        if (apiLevel <= 26) { // API level 26 is Android 8.0.
```

Comment:
I assume we should do this for api levels 23..26, since older api levels don't have such entity